public/app/shared/activityTimelineEventDetails.html

```diff
@@ -23,10 +28,30 @@ <h3 class="event-details-title">
     </h3>
 
     <p class="event-details-description">
-      <i class="fa fa-comment" data-ng-if="display.comment"></i>
-      <i class="fa fa-thumbs-up" data-ng-if="display.like"></i>
-      <i class="fa fa-eye" data-ng-if="display.view"></i>
-      <strong data-ng-bind="display.description"></strong>
+      <span data-ng-if="pageContext.tool === 'dashboard'"> 
+        <i class="fa fa-comment" data-ng-if="display.comment"></i>
+        <i class="fa fa-thumbs-up" data-ng-if="display.like"></i>
+        <i class="fa fa-eye" data-ng-if="display.view"></i>
+        <strong data-ng-bind="display.description"></strong>
+      </span>
+
+      <span data-ng-if="pageContext.tool === 'assetlibrary'"> 
+        <a tool-href
+           data-tool="dashboard"
+           data-course="pageContext.course"
+           data-id="activity.user.id"
+           data-referring-tool="pageContext.tool"
+           data-referring-id="pageContext.id"
+           class="event-details-strong"
+           data-ng-bind="activity.user.canvas_full_name"></a>
+        <span data-ng-switch="activity.type">
+          <span data-ng-switch-when="view_asset">viewed this asset.</span>
+          <span data-ng-switch-when="like">liked this asset.</span>
+          <span data-ng-switch-when="asset_comment">commented on this asset, </span>
```

Comment:
@johncrossman Per designs in https://jira.ets.berkeley.edu/jira/browse/COL-807, comments and only comments are followed by additional info in the form of comment text. I agree it looks funny hanging out this way in the template, but I wanted the comment-styling elements below to be accessible from either dashboard or asset-library configurations.